### DISCUSSION TITLE
Glench MIA?

### DISCUSSION THREAD

**Discussion by burkybang** [Glench/ExtPay#243] [General]
I have not received any email replies from @Glench over the past 2 months. I finally sent him an email last week asking if he's okay. Has anyone else been experiencing this? When's the last time you've heard from him?

**Comment by burkybang** [comment]
**Update:** @Glench is still alive ([or at least his Twitter account](https://x.com/Glench/status/1861541502058471874))

**Comment by Glench** [comment]
Hey @burkybang I'm so sorry to have not been responding to emails these past couple months. I had an extended health issue that was quite severe. Starting to feel better I think so I'll see what I can do.

Even though I couldn't respond to emails, ExtensionPay and ExtPay have still been working as-is just fine. It seems there's just a few things that people needed manual support for that I haven't been able to respond to. So I'll also look into adding features that developers can use themselves instead of needing manual intervention.

**Reply by burkybang** [reply]
I'm glad to hear you're starting to feel better. Are you able to respond to emails now? I need some help with removing free users. I sent an email last week.

**Comment by Cside** [comment]
@Glench 
Please take care of yourself.
I'm very grateful for ExtesionPay, it has helped me a lot.
